src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs

```diff
@@ -210,12 +210,24 @@ public void SelectCollectionAndClose(string path)
 			SelectedPath = path;
 			if (!string.IsNullOrEmpty(path))
 			{
+				if (IsInASourceCollectionFolder(path))
+				{
+					var msg = L10NSharp.LocalizationManager.GetString("OpenCreateCloneControl.InSourceCollectionMessage",
+						"This collection is part of your 'Sources for new books' which you can see in the bottom left of the Collections tab. It cannot be opened for editing.");
```

Comment:
Sorry, I didn't notice. There was a very parallel message about a possible issue with opening a collection in the same module with an ID that seemed reasonable, so I just added another item to it. Since I generally don't know what I don't know, it's hard to know when I should expand my search for options beyond the first one that looks reasonable.